README.md

```diff
@@ -4,25 +4,22 @@ Thanks to everyone that is starring, forking, writing issues, pull requesting an
 
 ![updatedgif45729805790](https://user-images.githubusercontent.com/81049050/150833862-0a828939-f267-4bd2-931f-79df55d51e28.gif)
 
-# Dependecies (all Python dependecies will automatically be installed):
-1.  mpv player https://mpv.io/  (Must be installed through a package manager)
-2.  youtube-dl https://github.com/ytdl-org  (Python dependency)
```

Comment:
`youtube-dl` is installed throught `setup.py` like any other, it doesn't need to be mentioned here.

if it does fail to install, that means the other libraries most likely did to and that's a bit of a bigger problem. in the off chance that `youtube-dl` is the only failed installation, i just added it to the troubleshooting section since it would most likely be rare and shouldn't be such a high focus of most users